test/integration-tests/resources/scenario-non-local-templates/0-tasks.yml

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
how come there are no tests for `https` protocol?

Reply:
Will add those for sure